app/src/main/java/net/squanchy/schedule/domain/view/Event.java

```diff
@@ -27,15 +27,16 @@ public static Event create(
             Optional<ExperienceLevel> experienceLevel,
             List<Speaker> speakers,
             Type type) {
+        Optional<String> placeOptional = Optional.fromNullable(place);        // TODO get Optional<Place> as type in here
         return new AutoValue_Event.Builder()
                 .id(eventId)
                 .numericId(numericEventId)
                 .dayId(dayId)
                 .startTime(startTime)
                 .endTime(endTime)
                 .title(title)
-                .place(Optional.fromNullable(place))
-                .placeVisibility(place.isEmpty() ? View.GONE : View.VISIBLE)
+                .place(placeOptional)
+                .placeVisibility(placeOptional.map(placeValue -> View.VISIBLE).or(View.GONE))
```

Comment:
This fixes a NPE crash (place is supposed to be optional in the model, but it's not in _our_ models, so here's the issue)